Sources/PrimerSDK/Classes/User Interface/ApplePayPresentationManager.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ApplePayPresentationManager.swift
+//  PrimerSDK
+//
+//  Created by Jack Newcombe on 23/05/2024.
+//
+
+import Foundation
+import PassKit
+
+protocol ApplePayPresenting {
```

Comment:
Do we have a strict naming convention that we follow? I would rather write ApplePayPresentationProtocol than use the present continuous tense in names. I would always add a Protocol suffix because I could distinguish between classes and interfaces when I read code. Just a thought... We should have our public API assessed because different naming conventions are used, for example, `Primer3DSSDKProviderProtocol` and `protocol BanksComponentErrorable`, which is also a delegate btw, so why not `BanksComponentErrorDelegate`... Now I am not strictly advocating any of the ideas, I would just rather make one to rule them all. WDYT? @NQuinn27 @jnewc 
We are going to release a new version 3.0 at one point think it is ok to have a breaking change that is just a different API. BTW some of the protocol names are the same as on Android but also very different in some cases.

<img width="650" alt="Screenshot 2024-05-23 at 23 12 17" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/404c5765-ac89-4c10-8883-4caceadd2e00">

Reply:
Agree, we are not consistent here. 
My preference is to not have `Protocol` in the protocol name, but rather be descriptive on what the protocol does. As @jnewc mentioned elsewhere, this is worth a decision and a standard put in place.